internal/cli/impl_tui.go

```
@@ -0,0 +1,376 @@
+		case compatMatches(msg, keymap.Switch):
+			m.resetLogScroll()
+			switch m.logTypeShow {
+			case logTypeShowAll:
+				m.logTypeShow = logTypeShowErr
+			case logTypeShowErr:
+				m.logTypeShow = logTypeShowOut
+			case logTypeShowOut:
+				m.logTypeShow = logTypeShowAll
+			}
+		case compatMatches(msg, keymap.LogUp):
+			// TODO: limit until first line
```

golangci-lint (1.62.2)

internal/cli/impl_tui.go:186: Line contains TODO/BUG/FIXME: "TODO: limit until first line"

(godox)